lib/PhasarLLVM/DataFlowSolver/IfdsIde/Problems/ExtendedTaintAnalysis/AbstractMemoryLocationFactory.cpp

```diff
@@ -394,12 +398,17 @@ AbstractMemoryLocationFactoryBase::withTransferToImpl(
 
           if (!From->offsets().empty()) {
             return std::next(AML->offsets().begin(),
-                             From->offsets().size() - 1);
+                             From->offsets().size() -
+                                 1); // FIXME @Fabian clang-tidy complains about
+                                     // narrowing conversion
```

Comment:
@MMory you should create an issue for this, otherwise this get's lost in the comments.

We always write it like this:
```
// FIXME (secure-software-engineering/phasar#1337): short desc
```
This allows IDE's to insert a link an bring one directly to the gh issue :)

Reply:
Fixed